.github/workflows/automerge-dependabot-webex-dependency-updates.yml

```diff
@@ -0,0 +1,23 @@
+name: "Dependabot Automerge"
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  dependency-automerge:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: automerge
+        if: github.event.pull_request.user.login == 'dependabot[bot]'
```

Comment:
won't this also merge other dependabot PRs?

Reply:
It will

Reply:
@lalli-flores are you ok with merging the other dependabot PRs, for third party dependencies, or should we find a way to identify the webex PRs specifically?